tests/unit/nodes/physics_engine/test_fluid_forces.py

```diff
@@ -0,0 +1,107 @@
+import math
+
+import numpy as np
+import pytest
+
+from boat_simulator.nodes.physics_engine.fluid_forces import MediumForceComputation
+
+
+@pytest.fixture
+def medium_force_setup():
+    lift_coefficients = np.array([[0, 0], [5, 0.57], [10, 1.10], [15, 1.39], [20, 1.08]])
+    drag_coefficients = np.array([[0, 0.013], [5, 0.047], [10, 0.144], [15, 0.279], [20, 0.298]])
+    areas = np.array([[0, 9.0], [45, 9.0], [90, 9.0], [135, 9.0], [180, 9.0]])
+    fluid_density = 1.225
+
+    computation = MediumForceComputation(
+        lift_coefficients, drag_coefficients, areas, fluid_density
+    )
+    return computation
+
+
+def test_initialization(medium_force_setup):
+    assert isinstance(medium_force_setup.lift_coefficients, np.ndarray)
+    assert isinstance(medium_force_setup.drag_coefficients, np.ndarray)
+    assert isinstance(medium_force_setup.areas, np.ndarray)
+    assert isinstance(medium_force_setup.fluid_density, (int, float))
+
+
+@pytest.mark.parametrize(
+    "apparent_velocity, orientation, expected_angle",
+    [
+        # Tests if apparent velocity is 0
+        (np.array([0, 0]), 0, 0),
+        (np.array([0, 0]), 45, 45),
+        (np.array([0, 0]), 90, 90),
+        (np.array([0, 0]), 180, 180),
+        (np.array([0, 0]), 270, 90),
+        (np.array([0, 0]), 360, 0),
+        (np.array([0, 0]), 450, 90),
+        # Tests if apparent velocity is not 0
+        (np.array([1, 0]), 0, 0),
+        (np.array([0, 1]), 0, 90),
+        (np.array([-1, 0]), 0, 180),
+        (np.array([0, -1]), 0, 90),
+        (np.array([1, 1]), 0, 45),
+        (np.array([1, -1]), 0, 45),
+        (np.array([-1, 1]), 0, 135),
+        (np.array([-1, -1]), 0, 135),
+        # Tests for apparent velocity other than unit vectors
+        (np.array([2, 0]), 0, 0),
+        (np.array([0, 2]), 0, 90),
+        (np.array([-2, 0]), 0, 180),
+        (np.array([0, -2]), 0, 90),
+        (np.array([2, 2]), 0, 45),
+        (np.array([2, -2]), 0, 45),
+        (np.array([-2, 2]), 0, 135),
+        (np.array([-2, -2]), 0, 135),
+        # Tests for orientation other than 0
+        (np.array([1, 0]), 45, 45),
+        (np.array([0, 1]), 45, 45),
+        (np.array([-1, 0]), 45, 135),
+        (np.array([0, -1]), 45, 135),
+        (np.array([1, 1]), 45, 0),
+        (np.array([1, -1]), 45, 90),
+        (np.array([-1, 1]), 45, 90),
+        (np.array([-1, -1]), 45, 180),
+    ],
+)
+def test_calculate_attack_angle(
+    apparent_velocity, orientation, expected_angle, medium_force_setup
+):
```

Comment:
Great test suite so far. A few suggestions for some more tests:
- Try using orientations that fall outside of the typical expected range (negative numbers, larger than 360, etc.)